### ISSUE TITLE
sops downloads aarch64 binary

### ISSUE THREAD

**Issue by bkero** [nklmilojevic/asdf-sops#2]
**Describe the bug**

The binary that this `install` script downloads does not indicate an architecture. That architecture is defaulting to aarch64 which is probably incorrect for most people. The script should detect the architecture and download the proper binary.

**Steps to reproduce**

```sh
$ uname -i
x86_64
$ asdf plugin add sops
$ asdf install sops latest
$ asdf set global sops latest
$ sops version
/home/bkero/code/asdf/lib/commands/command-exec.bash: line 28: /home/bkero/.asdf/installs/sops/3.7.2/bin/sops: cannot execute binary file: Exec format error
/home/bkero/code/asdf/lib/commands/command-exec.bash: line 28: /home/bkero/.asdf/installs/sops/3.7.2/bin/sops: Success

**Expected behavior**

The x86_64 (amd64) version of the binary should be downloaded

**Comment by nklmilojevic** [comment]
hey @bkero! The current problem is that latest sops binary is not correctly compiled for amd64. 

https://github.com/mozilla/sops/issues/1023

The [PR that fixes this was merged today](https://github.com/mozilla/sops/pull/1026), but it still hasn't been released. As soon as there is a new release (I suppose 3.7.3), this will automatically be fixed. I'm keeping this issue opened until it is resolved on the upstream.

**Comment by bkero** [comment]
Hi there, yes. I independently found the sops bug too. I suppose this bug could turn into a feature request at some point. I solved this by just using 3.7.1 until the bug is fixed. Thank you for the fast reply!

**Comment by nklmilojevic** [comment]
@bkero the issue is fixed in upstream now - I will make plugin better, this was just a simple c/p from my other asdf integration, skaffold :) Now everything works - let me know if something is not right.
